uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -379,12 +380,13 @@ define([
         }
 
         if (missingRequiredFields.length) {
-          var notification = CRM.alert(missingRequiredFields.join(', '),
-            missingRequiredFields.length === 1 ? 'Required field' : 'Required fields', 'error');
+          var notificationTitle = missingRequiredFields.length === 1 ? 'Required field' : 'Required fields';
+          var missingFields = missingRequiredFields.join(', ');
+          var alert = notification.alert(notificationTitle, missingFields);
 
-          notification && $timeout(function () {
-            notification.close();
-            notification = null;
+          alert && $timeout(function () {
+            alert.close();
```

Comment:
Why do we need to do this? `CRM.alert`supports a parameter for the fadeout time

> expires: how long to display this message before fadeout (in ms)
set to 0 for no expiration
defaults to 10 seconds for most messages, 5 if it has a title but no body, or 0 for errors or messages containing links